docs/actions.md

```diff
@@ -44,7 +44,7 @@ Review the following steps and examples to create your first JavaScript action.
 2. Create an action from the following JavaScript function. For this example, the action is called 'hello'.
 
   ```
```

Comment:
I already played adding this "shell" and also "bash" it didn't do anything to our syntax, and also cause some problems with highlighting that I can't remember now.